src/config-hardware/audio-apps.md

```diff
@@ -0,0 +1,39 @@
+# Audio Applications & Extensions
+
+These applications provide control and customization settings for audio devices beyond the `Settings` menu.
+
+---
+
+## EasyEffects
+
+[EasyEffects](https://github.com/wwmm/easyeffects) is a third-party application that can apply audio effects and filters to audio input and output streams. You can download and install EasyEffects from the Pop!\_Shop.
+
+![EasyEffects](/images/audio-mic/easyeffects.png)
+
+## Audio Output Switcher
+
+[Audio Output Switcher](https://extensions.gnome.org/extension/751/audio-output-switcher/) is a GNOME extension that adds selectable audio input/output devices to the System menu. See [this section](/customize-pop/gnome-tweaks-extensions/gnome-extensions.md) for more information about installing GNOME Extensions.
+
+![Audio Output Switcher](/images/audio-mic/output-switcher.png)
+
+## Alsamixer
+
+Alsamixer is a graphical command-line program that directly interfaces with the Advanced Linux Sound Architecture (ALSA) framework. Alsamixer provides more advanced audio configuration options, like selecting sound cards and viewing all system audio devices. Alsamixer comes preloaded in Pop!_OS and does not require installation.
+
+### Launching Alsamixer
+
+Open the Terminal and type `alsamixer` to launch the program.
+
+![Launch Alsamixer](/images/audio-mic/launch-alsamixer.png)
+
+### Alsamixer Controls
+
+Many Alsamixer functions are controlled using keyboard commands. Press `F1` to view all Alsamixer controls.
+
+![Alsamixer Keyboard Controls](/images/audio-mic/alsamixer-controls.png)
+
+### Selecting a Soundcard
```

Comment:
`Soundcard` is one word in the title but two words in the sentence below it and in the screenshot?